cypress/support/commands.ts

```diff
@@ -35,3 +38,18 @@ Cypress.Commands.add(
     });
   },
 );
+
+Cypress.Commands.add('patchVM', (vmName: string, status: string) => {
```

Comment:
We could be more specific about the `status` value, e.g. `status: 'Always' | 'Halted' | ...`

Reply:
it's in below commands `startVM` and `stopVM`.